app/Resources/views/historiccfa/historiccfa.html.twig

```diff
@@ -3,37 +3,207 @@
+                    </p>
+                    <p>
+                        Employeurs ! Pour nous aider à participer à la relève du "papy-boom" de demain, aidez-nous à former
+                        notre jeunesse aux métiers du social et médico-social afin de réussir leur devenir professionnel :
+                        embauchez des apprentis !
+                    </p>
+                    <p>
+                        Apprentis ! N'hésitez pas à nous contacter, nous vous offrirons notre meilleur accueil et un
+                        personnalisé pour rechercher le maître d'apprentissage qui facilitera votre insertion.
+                    </p>
+                    <p>
+                        Nous comptons sur votre enthousiasme et votre mobilisation et vous souhaitons une excellente visite sur
+                        notre site…
+                    </p>
+                </div>
+            </div>
+        </div>
+    </div>
+    <div class="row">
+            <h2 class="mb-5 style-subtitle col-md-10 offset-1 my-5">Le projet du CFA</h2>
+        <div class="col-md-10 offset-1 my-5">
+            <p>
+                Le projet du CFA est porteur d'un engagement qui s'inscrit dans la logique des orientations stratégiques
+                définies par la Région dans le domaine de l'apprentissage.
+            </p>
+            <p>
+                Dans le cadre du Contrat de Plan Régional de Développement des Formations Professionnelles (CPRDFP 2011-2014)
+                signé entre l'État et la Région Centre-Val de Loire le 5 juillet 2011, de la Convention quinquennale signée
+                le 17 juillet 2012 entre la Région et l'association gestionnaire du CFA, celle-ci a développé son projet
+                autour de trois axes :
+            </p>
+            <ul>
+                <li>le positionnement de l'offre de formation du CFA dans le dispositif global de formation professionnelle</li>
+                <li>la prise en charge de l'apprenti de l'amont à l'aval de la formation</li>
+                <li>le renforcement des partenariats avec les sites qualifiants.</li>
+            </ul>
+        </div>
+    </div>
 
+    <div class="row">
+        <div class="col-md-10 offset-1 my-5">
+            <div class="row justify-content-center">
+                <h3>L'histoire du CFA</h3>
+            </div>
+            <div class="row">
+                <div class="timeline">
                     {% for historicCfa in historicCfas %}
-
-                            <div class="timeline-item timeline-item-{% if historicCfa.id is even %} 'left' {% else %} 'right' {% endif %}">
-                                <div class="timeline-point timeline-point-primary">
-                                    <i class="fa fa-star"></i>
+                        <div class="timeline-item timeline-item-{% if historicCfa.id is even %} 'left' {% else %} 'right' {% endif %}">
```

Comment:
c'est plus lisible de faire une condition ternaire ici timeline-item-{{ historicCfa.id is even ? 'left' : 'right'  }}

Reply:
en effet c'est ce que tu avais écrit mais ça ne fonctionnait pas. Je te l'ai dit au moment où tu partais en me disant que je trouverai bien. Je n'ai trouvé que ça en solution. Mais je peux quand même rééssayer .

Reply:
ça ne fonctionne pas! ça n'alterne pas gauche droite..